### DISCUSSION TITLE
Addon UI credentials different from HA user credentials?

### DISCUSSION THREAD

**Discussion by NicoLeOca** [hassio-addons/addon-adguard-home#202] [General]
I just can't find which password I need to use to connect to the web UI directly.
My final purpose is to use this soft to sync 2 AGH instances:
https://github.com/bakito/adguardhome-sync
It requires to authenticate.
However my HA user credentials just don't work.
Also when I try to hit http://myinternalIP:80 directly, it refuses my credentials too.
This seems logical though.

Can you help please?

**Comment by sinclairpaul** [comment]
It is the Home Assistant credentials, assuming you have exposed the web interface on a port.

**Reply by NicoLeOca** [reply]
thanks for the answer
There is definitely an issue on my system
I suspect trusted networks messes things around

**Reply by NicoLeOca** [reply]
Yes, my trusted networks was not configured properly
I got rid of everything and I can connect to AGH directly now
